spec/models/comment_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :body }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :body }
+  end
+
+  describe 'Associations' do
+    it { is_expected.to belong_to :post }
```

Comment:
have you tested the association between user and comment in the `User` model spec?

Reply:
all associations now tested!